src/main/java/uk/gov/gchq/gaffer/doc/user/walkthrough/Traversing.java

```diff
@@ -179,4 +179,9 @@ public Traversing() {
         }
         return printableWalks;
     }
+
+    public static void main(final String[] args) throws OperationException, IOException {
+        final Traversing walkthrough = new Traversing();
+        walkthrough.run();
```

Comment:
What was missing from this?

Reply:
All the other walkthrough code classes have a main function, but for some reason this one didn't which prevented it from being run manually. It still worked for the build itself as that creates an object of this class elsewhere and doesn't call main.